barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.cpp

```diff
@@ -100,6 +100,133 @@ poly_triple serialize_arithmetic_gate(Program::Expression const& arg)
     pt.q_c = uint256_t(arg.q_c);
     return pt;
 }
+
+void assign_linear_term(mul_quad_<fr>& gate, int i, uint32_t w, fr const& scaling)
```

Comment:
Could you please add a description to the function and use more descriptive names for variables than "i" and "w"?

Reply:
Done